tests/robustness/coverage/README.md

```diff
@@ -0,0 +1,123 @@
+## Overview
+
+This is still Work In Progress.
```

Comment:
This doesn't help a reader in any way. The documented process should still work and bring value, we could list which parts return inaccurate results and propose improvements, but overall we should not merge unfinished work. This should be working minimal viable product.